samples/common/src/utils.rs

```diff
@@ -74,29 +71,34 @@ where
 /// Use Ibeji to discover the endpoint for a digital twin provider that satifies the requirements.
 ///
 /// # Arguments
-/// * `invehicle_digitial_twin_servuce_url` - In-vehicle digital twin service URL.
+/// * `invehicle_digitial_twin_servuce_uri` - In-vehicle digital twin service URI.
```

Comment:
Please change this comment and the parameter to `invehicle_digital_twin_service_uri`

Reply:
Fixed.